src/lib/parsers/Args.ts

```diff
@@ -14,7 +14,7 @@ import type {
 	User,
 	VoiceChannel
 } from 'discord.js';
-import type * as Lexure from 'lexure';
+import * as Lexure from 'lexure';
```

Comment:
If possible, can we just plugin `@sapphire/lexure` instead?

Reply:
I was going to do that in a separate PR, but I can do that in this one.

Reply:
please do if possible! That'd be excellent, but up to you.